src/sentry/workflow_engine/processors/detector.py

```diff
@@ -112,14 +112,19 @@ def get_detector_by_event(event_data: WorkflowEventData) -> Detector:
 
     issue_occurrence = evt.occurrence
 
+    detector: Detector | None = None
+
+    if issue_occurrence is None or evt.group.issue_type.detector_settings is None:
+        # if there are no detector settings, default to the error detector
+        detector = Detector.get_error_detector_for_project(evt.project_id)
```

Comment:
is this the correct logic for the error detector now?

for a feedback issue...  issue_occurrence would not be none (i think?), but the `detector_settings` would be `None`, then it would get the error detector and return it. (rather than falling through to the issue stream).